dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/ConnectionManagerMBean.java

```diff
@@ -81,20 +81,31 @@
  * It should be noted that the : character from the JNDI name is replaced by the . character because : is not valid in an object name.
  * </li>
  * </ul>
+ *
  */
 public interface ConnectionManagerMBean {
     /**
      * Purge the contents of the connection pool associated with
      * this Connection Manager.
-     * 
-     * @param doImmediately 'immediate' to purge the pool contents immediately,
-     *            otherwise 'normal'
+     *
+     * @param doImmediately The priority to be used to purge the connection pool.
+     *            Priority may be <code>"immediate"</code>, <code>"abort"</code> or <code>"normal"</code>.
+     *            Immediate purges the pool contents as quickly as possible but waits for transactions to complete,
+     *            abort purges the pool by aborting connections without waiting for transactions to complete and
+     *            normal purges the pool with normal priority.
      * @throws MBeanException
```

Comment:
Its optional since I don't believe we are documenting all of the behavior in java doc, but for Immediate we may want this,  "Immediate sets the total connection count to 0 while its purging the pool contents as quickly as possible but waits for transaction to complete"  By adding the reset of the total connection count, it implies that we temporarily can exceed maximum connections until all of the purged connections are destroyed.   

We may want to add the default is Normal.